draft-wood-key-consistency.md

```diff
@@ -148,55 +158,132 @@ to ensure consistency here might include, though are not limited to:
 The precise external mechanism used here depends largely on the threat model. If there is a trusted
 external log for keys, this may be a viable solution.
 
-## Proxy-Based Key Discovery {#proxy-based}
-
-In this model, there exists a proxy that fetches keys from servers on behalf of multiple users. If this
-proxy is trusted, then all users which request a key from this server are assured they have a consistent
-view of the server key. However, if this proxy is not trusted, operational risks may arise:
+## Single Proxy Discovery {#proxy-based}
+
+In this model, there exists a proxy that fetches keys from servers on behalf of multiple users, as shown
+below.
+
+~~~
+┌──────────┐
+│          │
+│  Client  ├──────────┐
+│          │          │
+└──────────┘          │
+                      │
+┌──────────┐         ┌▼─────────┐      ┌─────────┐
+│          │         │          │      │         │
+│  Client  ├─────────►  Proxy   --─────► Server  │
+│          │         │          │      │         │
+└──────────┘         └▲─────────┘      └─────────┘
+      x               │
```

Comment:
Maybe use an ellipsis (…) here instead?

Reply:
Eh, I left this as-is for now. We can clean it up later if it's bothersome to others.